EsraaAbdelrahem/movies-
# submitting movie info & use local storage to save the data at the browser

the reason of this pull request:
=======================
1- creating new component for movies submission.
2- make a simple form and use form validation that make the user enter name of move year and budget also we should use a local storage for this form to save data.

tools of usage:
===================
I didn't use any plugins at this component.

how to test:
=========================
try to enter values and check if its a valid input type or not.
